nvr/nvr.py

```diff
@@ -288,6 +288,11 @@ def parse_args(argv):
     parser.add_argument('-q',
             metavar = '<errorfile>',
             help    = 'Read errorfile into quickfix list and display first error.')
+    parser.add_argument('-r',
```

Comment:
I try to stay compatible to Vim's options (because some people use `nvr` as drop-in replacement for `vim`), so we should prefer long-style options for nvr-specific options.

`--setreg`? :)

Reply:
That sounds like a good idea. Do you want me to add a `getreg` too, and in that way turn nvr into a command-line clipboard with 26+2 registers? Or do you think that is out of scope for this project?